index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width='device-width', initial-scale=1.0">
```

Comment:
![err1](https://github.com/ammadCheema36/HelloMicroverse/assets/99656687/9cd3deab-f799-4b82-abdc-b86a37b6299d)


- [ ]  You have webhint errors in your submission. You should always fix these errors because it ensures that your code is well written. please change line  5 to 
`    <meta name="viewport" content="width=device-width, initial-scale=1.0">
` 😄